examples/compose-object-test-example.js

```diff
@@ -20,8 +20,8 @@ const os = require('os')
 const splitFile = require('split-file')
 const fs = require('fs')
 
-var Minio = require('../dist/main/minio')
-var Helpers = require('../dist/main/helpers')
+var Minio = require('minio')
+var Helpers = require('minio/dist/main/helpers')
```

Comment:
Do you want to start modifying the variable instance with `const` rather than `var`?

Reply:
I don't think it matters in example.

Reply:
I guess it would be a way to say that the library has been modernized. But it's okay if we're still using `var` on examples.

Reply:
`prefer-const` is previous disabled in eslint, I don't know how maintainers think.